### DISCUSSION TITLE
Getting rejected again...

### DISCUSSION THREAD

**Discussion by PRO-2684** [community/community#58592] [GitHub Education]
### Hi everyone,

I've tried to verify myself and had already changed my profile with my full name on it (although I hate to do so), but I still get rejected. My fellow students have passed the verification with screenshots, but I've been rejected again.

**Comment by Rxxbertx** [comment]
Same .... I dont know whats going on

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
